conf.py

```diff
@@ -75,7 +75,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = ['reference/technical_information/choose_database.rst']
```

Comment:
not sure to understand this option, what it does for real? still included in other pages but ignored in the search / index?